app/views/layouts/_matomo_tag.html.haml

```diff
@@ -1,15 +1,23 @@
+
+- if Spree::Config.matomo_tag_manager_url.present?
+  :javascript
+    var _mtm = _mtm || [];
+    _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    var u="#{Spree::Config.matomo_tag_manager_url}";
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u; s.parentNode.insertBefore(g,s);
 - if Spree::Config.matomo_url.present?
```

Comment:
I'm being a bit picky here, but, with haml's lack of curly braces etc., I think a blank line between the snippet above and the `if` would make things more readable. We'll then see at a glance that these are two separate snippets.

Reply:
np